procedures/accounts_setup.rst

```diff
@@ -0,0 +1,19 @@
+Here is a list of accounts and access needed to be successful in Release Engineering
+
+General:
+ * `JIRA <https://jira.mozilla.com/projects/RELENG/issues>`_ - add username to Mozilla instance
+ * `Commit Access <https://www.mozilla.org/en-US/about/governance/policies/commit/access-policy/>`_ - fill out form, file bug
+ * Bugzilla
+ * Phabricator
+ * Github
```

Comment:
Nit: worth a word on being added to @mozilla/releng group? We have others as well but it's a good start.